utils/ssh.py

```diff
@@ -480,6 +480,14 @@ def set_initial_file_end(self):
             fstat = sshtail._sftp_client.stat(self._remote_filename)
             self._remote_file_size = fstat.st_size  # Seed initial size of file
 
+    def raw_output(self):
+        """Returns the output of the log as one (raw) string"""
+        return '\n'.join(self.lines_as_list())
```

Comment:
1. You don't need to create a list for this. Just use the generator object:
    '\n'.join(line for line in self)
2. I don't like calling this raw_output as it is not raw. The newlines are gonna be mangled depending on the platform we are running on. The '\n' is <CR> on Mac, <LF> on unix and <CR><LF> on windows